docker-compose.yml

```diff
@@ -163,9 +163,9 @@ services:
       context: .
       dockerfile: Dockerfile.database
     expose:
-      - "3306"
+      - "3307"
```

Comment:
This doesn't look related to #1145

Could you revert the changes to this file?

You can keep them on your local, but we don't need to commit/land them.

Reply:
Sure,  let me correct this!